plugin/README.md

```diff
@@ -0,0 +1,3 @@
+WIP
```

Comment:
## The plugin framework

https://github.com/pingcap/tidb/blob/master/docs/design/2018-12-10-plugin-framework.md